com.unity.shadergraph/Editor/Data/Graphs/AbstractShaderProperty.cs

```diff
@@ -7,6 +7,36 @@ namespace UnityEditor.ShaderGraph.Internal
     [Serializable]
     public abstract class AbstractShaderProperty : ShaderInput
     {
+
+        public virtual int latestVersion { get; } = 0;
```

Comment:
This should be static and override-able, but I am having a brain fart on the way to do that